packages/core/src/utils.ts

```diff
@@ -243,3 +248,61 @@ export async function getRawConfiguration(
   }
   return sygmaConfig;
 }
+
+/**
+ * Validate Substrate address.
+ * @param {string} address
+ */
+export function isValidSubstrateAddress(address: string): boolean {
+  try {
+    encodeAddress(isHex(address) ? hexToU8a(address) : decodeAddress(address));
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
+
+/**
+ * Validate EVM address.
+ * @param {string} address
+ * @returns {boolean}
+ */
+export function isValidEvmAddress(address: string): boolean {
+  return !!ethers.utils.isAddress(address);
```

Comment:
does `isAddress`function throws as well?

Reply:
No, it returns true or false, if it returns false I throw an error from method "isValidAddressForNetwork"